chiaswap/main.py

```diff
@@ -160,13 +160,16 @@ def ui_get_amounts(input, prices):
     )
     print()
     xch_amount = Decimal(input("How much XCH is being traded? > "))
+    print("How much XCH fee (default: 0.00005 mojos)?")
+    print("(You should agree your fee with your counter-party. The fee will be used for either clawback, clean, or sweep spend.)")
+    fee_amount = Decimal(input("> ") or "0.00005")
```

Comment:
Adding another input asking for fee amount. The XCH side has to send `xch_amount + fee_amount` to create the coin with the `p2_delayed_or_preimage`.